site/docs/v1/tech/oauth/endpoints.adoc

```diff
@@ -2051,110 +2051,66 @@ This parameter is only returned during an HTTP redirect if it was provided on th
 [#client-secret-table]
 == Client Secret
 
-This table describes when the [field]#client_secret# is required.
-If required, it may be placed in either the `Authorization` header or the request body.
+This table can be used as a guideline for describing when the [field]#client_secret# is required.
 
-Even when the value is not required, if it is provided in the request, it must be valid.
+In each of the endpoint's documentation, it describes when and how to use the `client_secret` in the request body or the Authorization header using HTTP Basic Auth.
+
+Even when the client secret is not required, if it is provided it will be validated.
 
 Previous to version `1.28.0`, whether the client secret was required was controlled by the [field]#oauthConfiguration.requireClientAuthentication# value of the application object.
 In that case, a value of `true` is equivalent to a [field]#clientAuthenticationPolicy# of `Required` and a value of `false` is equivalent to a [field]#clientAuthenticationPolicy# of `NotRequired`.
 
 For versions `1.28.0` or later, use [field]#oauthConfiguration.clientAuthenticationPolicy# to configure this functionality.
 
 .When Is the Client Secret Required
-[cols="35,35,10,10,10"]
+[cols="35,35,15,15"]
 |===
-| Grant | clientAuthenticationPolicy value | PKCE Used | Secret Required in Header | Secret Required in Body
+| Grant | clientAuthenticationPolicy | PKCE Used | Client Secret Required
 
 |Authorization Code Grant
 |`Required`
-|N/A
-|Yes
-|No
-
-|Authorization Code Grant
-|`Required`
-|N/A
-|No
+|-
 |Yes
 
 |Authorization Code Grant
 |`NotRequired`
-|N/A
-|No
-|No
-
-|Authorization Code Grant
-|`NotRequiredWhenUsingPKCE`
-|Yes
-|No
+|-
 |No
 
 |Authorization Code Grant
 |`NotRequiredWhenUsingPKCE`
-|No
 |Yes
 |No
 
 |Authorization Code Grant
 |`NotRequiredWhenUsingPKCE`
 |No
-|No
 |Yes
 
 |Implicit Grant
-|N/A
-|N/A
-|N/A
-|N/A
-
-|Password Credentials Grant
-|`Required`
-|N/A
-|Yes
-|No
+|-
```

Comment:
I feel like the blanks are confusing here. I don't really know what they mean. I feel that the N/A was more descriptive

Reply:
I want to use `-`. This is a consistent pattern we have in docs and in the UI. If we want to add a footer that indicates `-`  implies not applicable we can do that.